tests/sles4sap/redirection_tests/ensa2_setup_Netweaver_WebGUI_access.pm

```diff
@@ -0,0 +1,156 @@
+        return_output => 'yes');
+
+    # Configure WebGUI services (via SQL statement)
+    # Userstore key check
+    assert_script_run('su - qesadm -c "hdbuserstore list"');
+    assert_script_run('su - qesadm -c "hdbsql -U DEFAULT \'select * from m_services\'" | tee');
+
+    # Activate WebGUI services (via SQL statement)
+    my @hdbsql = ();
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'MIMES\\' and ICFPARGUID = \\'0CMVCTDU211SGUZWM853S6A3S\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'MIMES\\' and ICFPARGUID = \\'1390BKBVEPQAYPVZ8O7GLZOJ9\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'UR\\' and ICFPARGUID = \\'0SLDKWIWJT0Y8UZ66DNUEDGQD\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'WEBGUI\\' and ICFPARGUID = \\'BO11PUMK7J2UU0LPKQWG0KGS7\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'SAP\\' and ICFPARGUID = \\'0000000000000000000000000\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'SAP\\' and ICFPARGUID = \\'9ROEQ0LR7N1DUVIJPWT3THCDZ\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'ITS\\' and ICFPARGUID = \\'0SLDKWIWJT0Y8UZ66DNUEDGQD\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'ITS\\' and ICFPARGUID = \\'7AEANJ9KQ34VA5EPCKEL8Z27R\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'BC\\' and ICFPARGUID = \\'0V000YHIHJTMAQZ31MI9AONBR\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'BC\\' and ICFPARGUID = \\'DFFAEATGKMFLCDXQ04F0J7FXK\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'GUI\\' and ICFPARGUID = \\'EEPI2GLFNOLHN7IW9R54I61RZ\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'PUBLIC\\' and ICFPARGUID = \\'DFFAEATGKMFLCDXQ04F0J7FXK\\'");
+    push(@hdbsql, "hdbsql -U DEFAULT update ICFSERVLOC SET ICFACTIVE = \\'X\\' where ICF_NAME = \\'DEFAULT_HOST\\' and ICFPARGUID = \\'FFFFFFFFFFFFFFFFFFFFFFFFF\\'");
+
+    foreach my $sql (@hdbsql) {
+        assert_script_run("sudo su - qesadm -c \"$sql\"");
+    }
+
+    # Restart NW instance again
+    my $output = sapcontrol(
+        webmethod => 'RestartInstance',
+        instance_id => $instance_data->[0]{instance_id},
+        sidadm => $instance_data->[0]{sap_sid},
+        return_output => 'yes');
+
+    disconnect_target_from_serial();
+
+    # Check URL is accessible from NW hosts (locally + remotely)
+    my %nw_hosts = %{$redirection_data->get_nw_hosts};
+    if (!%nw_hosts) {
```

Comment:
Yeah.. I think conditional schedule would be more fitting. 
It would allow you to remove `if' statements like this one.